Peer.Domain/Commands/Show.cs

```diff
@@ -16,30 +20,66 @@ public class Show
         private readonly ISorter<PullRequest>? _sorter;
         private readonly List<IFilter> _filters;
 
+        public ShowConfig Config { get; }
```

Comment:
I moved the interval from the types called options/args into a config type since we weren't actually allowing it to be overridden by options/args from the CLI and added the other settings for retry.